src/lib/containers/drug_upload_tool/SummaryTable.tsx

```diff
@@ -75,7 +76,14 @@ export function getFlatData(formData: any, steps: Step[]): SummaryFormat[] {
 }
 
 export default function SummaryTable(props: SummaryTableProps): JSX.Element {
-  const flatFormData = getFlatData(_.cloneDeep(props.formData), props.steps);
+  const [formData, setData] = useState(props.formData);
+  let flatFormData: SummaryFormat[] = [];
+  useEffect(() => {
```

Comment:
How come formData is being put in state?  I would think this is unnecessary since the component is only reading over formData